rpm/busybox.spec

```diff
@@ -465,6 +467,7 @@ rm -f %{buildroot}/bin/pidof
 %{_bindir}/whoami
 %{_bindir}/yes
 %{_sbindir}/chroot
+%{_sbindir}/seedrng
```

Comment:
Should we even provide this? It's not in `gnu-coreutils` at least.

Reply:
There's no equivalent as far as I know in coreutils. The closest thing is https://git.zx2c4.com/seedrng/about/.

Reply:
I removed the new programs from build